math/mathmore/src/GSLMultiRootFinder.cxx

```diff
@@ -158,7 +158,7 @@ const double * GSLMultiRootFinder::FVal() const {
 }
 const char * GSLMultiRootFinder::Name() const {
    // get GSL name
-   return (fSolver != 0) ? fSolver->Name().c_str() : "";
+   return (fSolver != 0) ? fSolver->Name().c_str() : nullptr;
```

Comment:
That should not be needed? That address actually has static storage duration as per [lex.string]. Are you sure you're solving a problem in the current code version? (Note that we *had* a problem here because `fSolver->Name()` returned a `std::string`; it now returns a `std::string&`.)